### PR TITLE
Implement InterviewerSlot, CandidateSlot, User, Role classes. Cleanup code.

### PR DESCRIPTION
**Pull request by grata-salve**
🔥 Remove unused lines in CandidateController.
✅ implement entities InterviewerSlot, CandidateSlot, User, enum Role.
🎨 refactor TimeSlot into InterviewerSlot.
:construction: temporarily comment out tests.

### CODE REVIEW

**Comment by CalculusEnjoyer** [comment]
|File|Coverage [28.57%]|:green_apple:|
|:-|:-:|:-:|
|[Role.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FRole.java)|100%|:green_apple:|
|[CoordinatorService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FCoordinatorService.java)|100%|:green_apple:|
|[CandidateController.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FCandidateController.java)|50%|:green_apple:|
|[Candidate.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FCandidate.java)|30%|:green_apple:|
|[InterviewerService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|28.57%|:green_apple:|
|[CandidateService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FCandidateService.java)|28.57%|:green_apple:|
|[Interviewer.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FInterviewer.java)|21.43%|:green_apple:|
|[Coordinator.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FCoordinator.java)|21.43%|:green_apple:|
|[CandidateSlot.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FCandidateSlot.java)|9.09%|:x:|
|[User.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FUser.java)|9.09%|:x:|
|[InterviewerSlot.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FInterviewerSlot.java)|5.66%|:x:|
|[InterviewerTimeSlot.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/b18733bbceae41b9cf572e794f5797aa20ab9063/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerTimeSlot.java)|0%|:x:|

|Total Project Coverage|19.57%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/services/InterviewerTimeSlot.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.intellias.intellistart.interviewplanning.services;
+
+/**
+ * Interviewer time slot class.
+ */
+public class InterviewerTimeSlot {
```
Comment (Archasmiel) [inline src/main/java/com/intellias/intellistart/interviewplanning/services/InterviewerTimeSlot.java:6]: Interviewer time slot in services?

**Comment by CalculusEnjoyer** [comment]
|File|Coverage [40.52%]|:green_apple:|
|:-|:-:|:-:|
|[Role.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2Fenums%2FRole.java)|100%|:green_apple:|
|[Status.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2Fenums%2FStatus.java)|100%|:green_apple:|
|[CoordinatorService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FCoordinatorService.java)|100%|:green_apple:|
|[CandidateController.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FCandidateController.java)|50%|:green_apple:|
|[Candidate.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FCandidate.java)|30%|:green_apple:|
|[InterviewerService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|28.57%|:green_apple:|
|[CandidateService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FCandidateService.java)|28.57%|:green_apple:|
|[Interviewer.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FInterviewer.java)|21.43%|:green_apple:|
|[Coordinator.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FCoordinator.java)|21.43%|:green_apple:|
|[CandidateSlot.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FCandidateSlot.java)|9.09%|:x:|
|[User.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FUser.java)|9.09%|:x:|
|[InterviewerSlot.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/87695ddc32bcd6e2764e7b151bf63aba84a934b6/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FInterviewerSlot.java)|4.76%|:x:|

|Total Project Coverage|29.26%|:green_apple:|
|:-|:-:|:-:|

**Review by Archasmiel — APPROVED** [review]
OK

**Pull request merged**